### PR TITLE
[sysdig][sysdig-deploy] add deprecation note to install text

### PR DESCRIPTION
**Pull request by achandras**
## What this PR does / why we need it:

In preparation for moving to the new sysdig-deploy chart, users should be notified that this chart is being deprecated.

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped

### CODE REVIEW

**Comment by achandras** [comment]
@ivanevsyukov please let me know what you think of the wording

**Comment by achandras** [comment]
This is what it looks like when you do an install/upgrade:
```
λ  charts/charts/sysdig sysdig-deprecation-note ✗ helm upgrade -n sysdig-agent sysdig-agent-1 ./ -f values.deploy.yaml --install
Release "sysdig-agent-1" has been upgraded. Happy Helming!
NAME: sysdig-agent-1
LAST DEPLOYED: Wed May 11 11:49:32 2022
NAMESPACE: sysdig-agent
STATUS: deployed
REVISION: 5
TEST SUITE: None
NOTES:
The agent for Sysdig Secure DevOps Platform is spinning up on each node in your
cluster. After a few seconds, you should see your hosts appearing in the
Sysdig Agent Health & Status Dashboard:

  https://app.sysdigcloud.com/#/dashboard-template/view.sysdig.agents?last=10

  https://secure.sysdig.com/#/events/l:600/*/*?viewAs=list


-------------------
DEPRECATION WARNING
-------------------

This chart is being deprecated on 2023-01-01 and will no longer receive any
updates. Please migrate to https://charts.sysdig.com/charts/sysdig-deploy as
soon as possible for new features and continued support.
```

**Comment by ivanevsyukov** [comment]
Minor changes to the deprecation warning:

This chart is being deprecated on 2023-01-01, at which point it will no longer receive any updates. Please migrate to https://charts.sysdig.com/charts/sysdig-deploy as soon as possible for new features, security updates, and continued support.

**Comment by achandras** [comment]
Updated!

**Review by yashgiri — APPROVED** [review]

**Review by lilx1ao — APPROVED** [review]

File: charts/sysdig/templates/NOTES.txt [hunk 1]
```diff
@@ -6,4 +6,11 @@ Sysdig Agent Health & Status Dashboard:
 
   https://secure.sysdig.com/#/events/l:600/*/*?viewAs=list
 
-No further action should be required.
+
+-------------------
+DEPRECATION WARNING
+-------------------
+
+This chart is being deprecated on 2023-01-01, at which point it will no longer
```
Comment (lilx1ao) [inline charts/sysdig/templates/NOTES.txt:14]: farewell 👋

**Pull request merged**